components/Sidebar/Sidebar.jsx

```diff
@@ -8,7 +8,8 @@ const Sidebar = () => {
       <SidebarLogo />
       <nav>
         <ul className="flex flex-col gap-1">
-          <SidebarItem name="Stations" to="/" />
+          <SidebarItem name="Dashboard" to="/" />
```

Comment:
I need to show you how to extract all the string into translation file so we can have a french version